velox/exec/RowContainer.cpp

```diff
@@ -466,11 +483,26 @@ void RowContainer::freeRowsExtraMemory(
   numRows_ -= rows.size();
 }
 
-void RowContainer::invalidateColumnStats() {
-  if (rowColumnsStats_.empty()) {
-    return;
+void RowColumn::Stats::removeCellSize(
```

Comment:
```
s/removeCellSize/removeOrUpdateCellStats?
```

Reply:
I tend to keep the name as it is only for removing the cell with specified size. How it is implemented is independent. Naming it "UpdateCellStats" is too broad for this case.

Reply:
it is either remove or update? So let's call it removeOrUpdateCellStats which is more accurate? you update the null count as well?